portal/src/main/scala/cromwell/pipeline/controller/ProjectController.scala

```diff
@@ -0,0 +1,53 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives.{ as, complete, delete, entity, get, onComplete, path, pathPrefix, _ }
+import akka.http.scaladsl.server.PathMatchers.JavaUUID
+import akka.http.scaladsl.server.Route
+import cromwell.pipeline.datastorage.dto.{ ProjectAdditionRequest, ProjectId }
+import cromwell.pipeline.service.ProjectService
+import cromwell.pipeline.utils.auth.AccessTokenContent
+import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport._
+
+import scala.concurrent.ExecutionContext
+import scala.language.{ implicitConversions, postfixOps }
+import scala.util.Success
+
+class ProjectController(projectService: ProjectService)(implicit executionContext: ExecutionContext) {
+
+  val route: AccessTokenContent => Route = accessToken =>
+    pathPrefix("projects") {
+      post {
+        entity(as[ProjectAdditionRequest]) { request =>
+          onComplete(projectService.addProject(request)) {
+            case Success(projectId) => complete(projectId.value)
```

Comment:
No need to use `.value` if you provide an appropriate formatter (see comment below).

Reply:
fixed